worth2/templates/base.html

```diff
@@ -55,27 +58,31 @@
     </ul>
 
     <ul class="nav navbar-nav navbar-right">
-        {% if request.user.profile.is_participant %}
+        {% if user.profile.is_participant %}
```

Comment:
Nevermind -- I think that check is valid. I could make it a little more clear, but anonymous users need to be able to see the navbar, in order to `/about/`, `/contact/`, and just for visual purposes.

This avatar change won't break for anonymous users, because an anonymous user isn't considered a Participant.